capellambse/helpers.py

```diff
@@ -367,6 +367,40 @@ def repair_html(markup: str) -> markupsafe.Markup:
     markup
         The repaired markup.
     """
+
+    def cb(node: etree._Element) -> None:
+        for k in list(node.keys()):
+            if ":" in k:
+                del node.attrib[k]
+
+    return handle_html(markup, cb)
+
+
+def handle_html(
+    markup: str, node_handler: cabc.Callable[[etree._Element], None]
+) -> markupsafe.Markup:
```

Comment:
When I worked on this with you, I intended these to be preliminary names so we could concentrate on getting the code working and not waste time thinking about what to name this function that hasn't even been written yet.

`handle_html` says basically nothing, and I'm also not really happy with the `node_handler` argument name. What do you think about this?

```suggestion
def process_html_fragments(
    markup: str, node_callback: cabc.Callable[[etree._Element], None]
) -> markupsafe.Markup:
```